core/API/DataTableManipulator/ReportTotalsCalculator.php

```diff
@@ -107,6 +108,10 @@ protected function manipulateDataTable($dataTable)
         /** @var DataTable\Row $totalRow */
         $totalRow = null;
         foreach ($firstLevelTable->getRows() as $row) {
+            if ($row->getMetadata('is_aggregate') == '1') {
+                continue; // skip aggregated row added by flattening
```

Comment:
Note: Before the flattening was applied after the totals row, so this wasn't needed. Now the flattener is applied before, and if `include_aggregate_rows` is set it add additional aggregation rows. Those rows need to be skipped for summing up values.